ui/src/test/webapp/app/query/query-component-spec.js

```diff
@@ -664,26 +664,138 @@ describe('The Query component', function() {
                 });
             });
 
-            describe('When adding seeds', function() {
+            describe('When adding seed pairs', function() {
```

Comment:
Can we keep the normal seed tests in too?

Reply:
normal tests are just below in the file